entity-service/src/test/kotlin/com/egm/stellio/entity/service/EntityServiceTests.kt

```diff
@@ -548,6 +580,99 @@ class EntityServiceTests {
         confirmVerified()
     }
 
+    @Test
+    fun `it should create a property instance with a default datasetId`() {
```

Comment:
I think those 2 tests do not bring a lot. Basically you are just testing the constructor of a Property object ...